src/components/colorPicker/ColorPalette.api.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "ColorPalette",
+  "category": "form",
+  "description": "A component for displaying a color palette",
+  "notes": "This is a screen width component",
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/ColorPickerScreen.tsx",
+  "images": [
+    "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/ColorPalette/ColorPalette.gif?raw=true"
+  ],
+  "props": [
+    {"name": "colors", "type": "string[]", "description": "Array of colors to render in the palette"},
+    {"name": "value", "type": "string", "description": "The value of the selected swatch"},
+    {
+      "name": "usePagination",
+      "type": "boolean",
+      "description": "Whether to use pagination when number of colors exceeds the number of rows",
+      "default": "true"
+    },
+    {
+      "name": "loop",
+      "type": "boolean",
+      "description": "Whether the colors pagination scrolls in a loop",
+      "default": "true"
+    },
+    {"name": "numberOfRows", "type": "number", "description": "The number of color rows from 2 to 5", "default": "3"},
+    {
+      "name": "animatedIndex",
+      "type": "number",
+      "description": "The index of the item to animate at first render",
+      "note": "Default is last"
```

Comment:
looks like `default` is a better key here

Reply:
No. "last" is not a value, it means the last item in the array.